### PR TITLE
Document the repository signing v4.9.0 service index resource

### PR DESCRIPTION
**Pull request by loic-sharma**
Part of https://github.com/NuGet/Engineering/issues/1778

### CODE REVIEW

**Comment by karann-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/api/repository-signatures-resource.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loshar-reposigning49/docs/api/repository-signatures-resource.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/api/repository-signatures-resource?branch=pr-en-us-1204) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C3%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201812032206115435-1204%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=I%2FKtjvy3SLnAIFIEy8dn8lwnk5Fe5QcROJGz9ihYG90%3D&st=2018-12-03T22%3A02%3A09Z&se=2019-01-03T22%3A07%3A09Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/api/repository-signatures-resource.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loshar-reposigning49/docs/api/repository-signatures-resource.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/api/repository-signatures-resource?branch=pr-en-us-1204) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C3%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201812032211557309-1204%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=QC8RqBzhYkqVkxuoKW8AMKaE0265aAJVmOrFuDLcRJ4%3D&st=2018-12-03T22%3A07%3A53Z&se=2019-01-03T22%3A12%3A53Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/api/repository-signatures-resource.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loshar-reposigning49/docs/api/repository-signatures-resource.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/api/repository-signatures-resource?branch=pr-en-us-1204) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C3%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201812032212337037-1204%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Ipokp5so0Lu0yxiqi5LeSS5iUIwZxBu8BRyVkG31w2g%3D&st=2018-12-03T22%3A08%3A51Z&se=2019-01-03T22%3A13%3A51Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/api/repository-signatures-resource.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loshar-reposigning49/docs/api/repository-signatures-resource.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/api/repository-signatures-resource?branch=pr-en-us-1204) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C3%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201812032216327074-1204%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=OaqXAJQTEPkbQtxdn5zELvGUhTUFjh8%2BvACFov19Fo0%3D&st=2018-12-03T22%3A12%3A31Z&se=2019-01-03T22%3A17%3A31Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/api/repository-signatures-resource.md [hunk 1]
```diff
@@ -63,15 +61,19 @@ The following request fetches the repository signatures index.
...
+allRepositorySigned | boolean          | yes      | Must be `false` on 4.7.0 resource
+signingCertificates | array of objects | yes      | 
 
 The `allRepositorySigned` boolean is set to false if the package source has some packages that have no repository
 signature. If the boolean is set to true, all packages available on the source must have a repository
 signature produced by one of the signing certificates mentioned in `signingCertificates`.
 
+> [!Warning]
+> The `allRepositorySigned` boolean should be false on the 4.7.0 resource. NuGet v4.7 and v4.8 clients cannot install packages
+> from sources that have `allRepositorySigned` set to true.
```
Comment (loic-sharma) [inline docs/api/repository-signatures-resource.md]: See https://github.com/NuGet/NuGet.Client/pull/2471 and https://github.com/NuGet/NuGet.Client/pull/2468. I didn't list these issues in the documentation as I didn't think they are particularly helpful

Comment (joelverhagen) [reply docs/api/repository-signatures-resource.md]: I would use "must" instead of "should"

**Comment by karann-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/api/repository-signatures-resource.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/loshar-reposigning49/docs/api/repository-signatures-resource.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/api/repository-signatures-resource?branch=pr-en-us-1204) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C3%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201812032238586832-1204%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=owvttjVh5VR8lzKMoPhUP2oYDou48lE6hmfJjGLjOa8%3D&st=2018-12-03T22%3A34%3A54Z&se=2019-01-03T22%3A39%3A54Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by loic-sharma** [comment]
@rido-min Personally I like that comment. It gives a clear example of when you should remove a certificate from the list.

**Pull request merged**